config/locales/en.yml

```diff
@@ -119,12 +119,50 @@ en:
     osbbs:
       osbb-name: "OSBB's name: "
       osbb-phone: 'Phone number: '
-      osbb-email: 'E-mail: ' 
+      osbb-email: 'E-mail: '
       osbb-website: "OSBB's Website: "
       btn-show: 'Show'
 
   account:
+    users:
+      show:
+        title: 'User info:'
+        user-name: 'User:'
+        user-birth: 'Date of Birth:'
+        user-email: 'Email:'
+        user-phone: 'Phone:'
+        user-sex: 'Sex:'
+        btn-edit: 'Edit User'
+        btn-delete: 'Delete User'
+        btn-back: 'Back to Users'
+      edit:
+        title: 'Edit User'
+        btn-edit: 'Edit User'
+        btn-cancle: 'Cancel'
+      form:
+        sex:
+          male: 'Male'
+          female: 'Female'
+          no_sex: 'not set'
+        first-name: 'First Name'
+        placeholder-first-name: 'Enter first name'
+        last-name: 'Last Name'
+        placeholder-last-name: 'Enter last name'
+        user-birth: 'Date of Birth: '
```

Comment:
do you need here extra space?

Reply:
![image](https://user-images.githubusercontent.com/46384633/96178881-b689c700-0f38-11eb-870a-69e1b582a264.png)

Reply:
fixed